python/tvm/tools/debug/ui/curses_widgets.py

```diff
@@ -0,0 +1,252 @@
+"""Widgets for Curses-based CLI."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import curses
+from tvm.tools.debug.ui import ui_common
+
+RL = ui_common.RichLine
+EXIT_TEXT = "exit"
+HOME_TEXT = "HOME"
+HELP_TEXT = "help"
+TVM_DBG_BOX_BOTTOM = "-----------------"
+NAVIGATION_MENU_COLOR_ATTR = "white_on_black"
+
+class NavigationHistoryItem(object):
+    """Individual item in navigation history."""
+
+    def __init__(self, command, screen_output, scroll_position):
+        """Constructor of NavigationHistoryItem.
+
+        Args:
+          command: (`str`) the command line text.
```

Comment:
Should use this document style: https://docs.tvm.ai/contribute/document.html#document-python